src/toil/jobStores/abstractJobStore.py

```diff
@@ -837,7 +837,7 @@ def getEmptyFileStoreID(self, jobStoreID=None):
         raise NotImplementedError()
 
     @abstractmethod
-    def readFile(self, jobStoreFileID, localFilePath):
+    def readFile(self, jobStoreFileID, localFilePath, symlink=False):
```

Comment:
@DailyDreaming Yeah, what is the symlink option supposed to do? I'm looking at #2570 and trying to see if it will break anything to symlink by default, and the lack of any documentation here is not helpful.